dashboard/test/factories/curriculum_factories.rb

```diff
@@ -0,0 +1,13 @@
+FactoryGirl.define do
+  factory :reference_guide do
+    association :course_version
+
+    sequence(:key, 1) {|c| "bogus-reference-guide-#{c}"}
```

Comment:
this is a bit confusing -- `1` tells it to use numbers, but `c` suggests a char. also `1` is the default. so I'd suggest:
```suggestion
    sequence(:key) {|n| "bogus-reference-guide-#{n}"}
```
or, if you want characters, then:
```
    sequence(:key, 'a') {|c| "bogus-reference-guide-#{c}"}
```